pandas/core/base.py

```diff
@@ -15,6 +16,24 @@
                             duplicated='IndexOpsMixin')
 
 
+def _validate_sorter(obj, sorter):
+    """
+    Checks whether or not the implementation of 'searchsorted'
+    for a given class instance 'obj' should accept a non-None
+    'sorter' argument, if one is provided. Raises a
+    NotImplementedError if that is not the case.
+    """
```

Comment:
I'm not sure why I would want to call `super`.  Doesn't that mean I end doing search twice?